src/psyclone/tests/domain/lfric/algorithm/lfric_alg_psyir_test.py

```diff
@@ -0,0 +1,140 @@
+# -----------------------------------------------------------------------------
```

Comment:
Please fix pylint unidiomatic-checks issues in this file.

Reply:
fixed.

Reply:
There are still pylint unidiomatic-checks issues

Reply:
Oops. Sorry, not sure how I missed them.